packages/mdx-embed/src/components/MDXEmbedProvider/index.ts

```diff
@@ -1 +1 @@
-export { MDXEmbedProvider } from './MDXEmbedProvider';
+export { MDXEmbedProvider } from './MdxEmbedProvider';
```

Comment:
Hmmm, this is gonna be a problem. I'd like to keep it as `MDXProvider`... can you revert this change? or does that mean you get errors?

Reply:
Yeah, I get errors when it's the original casing

Reply:
This should be resolved with #15